src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/use-trips-query.ts

```diff
@@ -304,3 +312,56 @@ function filterDuplicateTripPatterns(
     return true;
   });
 }
+
+function useJourneySearchModes(
+  defaultValue: InputMaybe<StreetMode>,
+): Types.Modes {
+  const {
+    preferences: {
+      flexibleTransport,
+      useFlexibleTransportOnAccessMode,
+      useFlexibleTransportOnDirectMode,
+      useFlexibleTransportOnEgressMode,
+    },
+  } = usePreferences();
+
+  const {
+    enable_flexible_transport,
+    use_flexible_on_accessMode,
+    use_flexible_on_directMode,
+    use_flexible_on_egressMode,
+  } = useRemoteConfig();
+
+  // If is DEV or QA env prioritizes local configuration
+  if ((!!JSON.parse(IS_QA_ENV || 'false') || __DEV__) && !!flexibleTransport) {
```

Comment:
I don't think this should be necessary. Also us "debug-users" want to be able to override these values in prod.